app/models.py

```diff
@@ -480,6 +480,15 @@ class ReplyToInbox(db.Model):
     updated_at = db.Column(db.DateTime, nullable=True, onupdate=datetime.datetime.utcnow)
 
 
+class Session(db.Model):
+    __tablename__ = "sessions"
+    id = db.Column(db.Integer, primary_key=True, nullable=False)
+    session_id = db.Column(db.String, nullable=False, index=True)
+    data = db.Column('data', db.Text, nullable=False)
+    created_at = db.Column(db.DateTime, nullable=False, default=datetime.datetime.utcnow)
+    updated_at = db.Column(db.DateTime, nullable=True, index=True, onupdate=datetime.datetime.utcnow)
```

Comment:
No FKs between this and any other table. How is this being populated/de-populated? I see the `session_id` is not unique.